### PR TITLE
Make LargeAddressAware an MSBuild task

### LINKED ISSUE #2: MSBuild task assembly
Can I make this into a proper MSBuild task DLL?  Does it need to be an executable?

### PR DESCRIPTION
**Pull request by jeffkl**
* Get rid of the .nuspec in favor of the new Pack target
* Only run the target if PlatformTarget is "x86" and OutputType is "Exe"
* Log a message so that users know it happened
* Rev version to 1.0.2

Closes #2

### CODE REVIEW

**Comment by jeffkl** [comment]
Target name and property `LargeAddressAware` stay the same.  Let me know what you think of the new conditions for only `Exe` projects that are `x86`.

**Comment by jeffkl** [comment]
Why aren't you using AppVeyor?

**Comment by KirillOsenkov** [comment]
:+1: all looks great :) A complete rewrite ;)

I'm not using AppVeyor because I was too lazy to set it up :) No other reasons. Never expected the project to change ever again :)

**Comment by KirillOsenkov** [comment]
I guess I forgot to upload the NuGet package, didn't I? :)

https://www.nuget.org/packages/LargeAddressAware/1.0.2

File: SetLargeAddressAware/LargeAddressAware.targets [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    To run this target, set the property LargeAddressAware to true in your project.
+    <PropertyGroup>
+      <LargeAddressAware>true</LargeAddressAware>
+    </PropertyGroup>
+    
+    The target only runs after CoreCompile so it is incremental and does not need Inputs/Outputs
+  -->
+  <Target Name="SetLargeAddressAware"
+          AfterTargets="CoreCompile"
+          Condition=" '$(LargeAddressAware)' == 'true' And Exists('$(LargeAddressAwareAssemblyFile)') And '$(OutputType)' == 'Exe' And '$(PlatformTarget)' == 'x86' ">
```
Comment (rainersigwald) [inline SetLargeAddressAware/LargeAddressAware.targets:21]: I think `OutputType=winexe` would be valid here as well.

Comment (KirillOsenkov) [reply SetLargeAddressAware/LargeAddressAware.targets:21]: Oh yeah, good catch!

Comment (KirillOsenkov) [reply SetLargeAddressAware/LargeAddressAware.targets:21]: Does MSBuild support parentheses? How would I change this expression?

`And ('$(OutputType)' == 'Exe' Or '$(OutputType)' == 'winexe') And `

would that work?

Comment (rainersigwald) [reply SetLargeAddressAware/LargeAddressAware.targets:21]: Yeah, we support nested parens in logical expressions. It's basically the one part of MSBuild that has a real parser!

Comment (KirillOsenkov) [reply SetLargeAddressAware/LargeAddressAware.targets:21]: lol I wouldn't call that a real parser :) I've seen some of that code :)

Comment (KirillOsenkov) [reply SetLargeAddressAware/LargeAddressAware.targets:21]: How's this? https://github.com/KirillOsenkov/LargeAddressAware/commit/909a19781b5c70d1ab9f9e0cb467a28e60579504

Comment (rainersigwald) [reply SetLargeAddressAware/LargeAddressAware.targets:21]: 👍

**Pull request merged**